content/trial_getting_started/introduction_to_blueprints/writing_blueprint.md

```diff
@@ -132,26 +128,24 @@ The [node template section](https://docs.cloudify.co/latest/developer/blueprints
 
-### Node definition:
-
-
-The TOSCA based node definition is based on an object oriented approach. As with other object oriented languages it includes interfaces, properties , and it also supports inheritance.
+## Node definition
 
+The TOSCA-based node definition is based on an object oriented approach to modeling resources. As with other object oriented languages it includes interfaces, properties, and supports inheritance.
 
 {{< highlight  yaml >}}
 http_web_server:
     type: cloudify.nodes.WebServer
 {{< /highlight  >}}
 
+This example defines a node named `http_web_server` of [type](https://docs.cloudify.co/latest/developer/blueprints/spec-node-types/) `cloudify.nodes.WebServer`. The type definition is derived from the import section. In this case we chose one of the [built in types](https://docs.cloudify.co/latest/developer/blueprints/built-in-types/). The `cloudify.nodes.WebServer` type is defined in the [types.yaml](https://cloudify.co/spec/cloudify/6.3.0/types.yaml) file.
 
-In this specific case we chose node name - _httpe_web_server_ which is of [type ](https://docs.cloudify.co/latest/developer/blueprints/spec-node-types/)_cloudify.nodes.WebServer_ - the type definition is derived from the import section. In this case we chose one of the [built in types](https://docs.cloudify.co/latest/developer/blueprints/built-in-types/) cloudify.nodes.WebServe_r which is defined in the [types.yaml](https://cloudify.co/spec/cloudify/6.2.0/types.yaml). (A simplified way to browse through the definition of each available node types in each plugin would be to use the IDE code completion feature which is supported through Cloudify [IDE Integration](https://docs.cloudify.co/latest/developer/ide_autocomplete/))
+> Note: A simplified way to browse through the definition of each available node types in each plugin would be to use the IDE code completion feature which is supported through the {{< param product_name >}} [IDE Integration](https://docs.cloudify.co/latest/developer/ide_autocomplete/))
 
-Most of the node types are derived from cloudify.nodes.Root which as you can see below defines a few interfaces , lifecycle, validation and monitoring. You can also see that operations can be specified for many different parts of a node's lifecycle. These operations offer you flexibility when modeling your environment. For example, a web server might be installed during the "create" operation, and the server software may be stopped via the "stop" operation.
+Most node types are derived from `cloudify.nodes.Root`, which defines a few interfaces for lifecycle, validation, and monitoring (see below). You can also see that operations can be specified for many different parts of a node's lifecycle. These operations offer you flexibility when modeling your environment. For example, a web server might be installed during the "create" operation, and the server software may be stopped via the "stop" operation.
```

Comment:
```suggestion
Most node types are derived from `cloudify.nodes.Root`, which defines a few interfaces for lifecycle, validation and monitoring (see below). You can also see that operations can be specified for many different parts of a node's lifecycle. These operations offer you flexibility when modeling your environment. For example, a web server might be installed during the "create" operation, and the server software may be stopped via the "stop" operation.
```

Reply:
I personally prefer the use of the Oxford comma, unless you have a particularly strong objection.